### PR TITLE
Use self hMargin negated instead of -3 in ScrollPane

### PR DESCRIPTION
**Pull request by greedy**

### CODE REVIEW

**Comment by MarcusDenker** [comment]
failing test on CI not related ( testBenchFor, I guess virtualised slave was too slow)

File: src/Morphic-Widgets-Scrolling/ScrollPane.class.st [hunk 1]
```diff
@@ -733,7 +733,7 @@ ScrollPane >> scrollBy: delta [
 		ifTrue: [scrollBar value: 0.0]
 		ifFalse: [scrollBar value: newYoffset asFloat / r].
 	(r := self hLeftoverScrollRange) = 0
-		ifTrue: [hScrollBar value: -3.0]
+		ifTrue: [hScrollBar value: (self hMargin negated asFloat)]
```
Comment (jecisc) [inline src/Morphic-Widgets-Scrolling/ScrollPane.class.st]: Is this useful? I see in ScrollBar>value: that the value should be between 0.0 and 1.0. (There is a #min:max: to ensure it)

Comment (greedy) [reply src/Morphic-Widgets-Scrolling/ScrollPane.class.st]: I think you're right. It seems the original code was in error and I changed it to the margin without considering that.

**Comment by estebanlm** [comment]
ouch, I accepted it and I didn't realise it lacks an issue number :(

**Comment by jecisc** [comment]
@estebanlm Related issue: https://pharo.fogbugz.com/f/cases/21578/Left-over-hardcoded-3-in-ScrollPane-should-be-self-hMargin

**Comment by greedy** [comment]
Oops. I forgot to link the fogbugz issue. The one @jecisc linked is indeed the one this pull request goes with.

**Pull request merged**